pkg/execution/queue/item.go

```diff
@@ -503,6 +503,20 @@ func GetEdge(i Item) (*PayloadEdge, error) {
 	}
 }
 
+// OptimizedParallelism returns true if the item has optimized parallelism
+// enabled.
```

Comment:
A note of the intention to default to `true` would be good here (like the `opcode.go` version).